### PR TITLE
Fix errors in php 8.0 (tested ingame)

### PR DESCRIPTION
**Pull request by JoshuaPHYT**

### CODE REVIEW

File: src/Jack/Bounty/EventListener.php [hunk 1]
```diff
@@ -224,7 +224,7 @@ public function handleCommand(CommandSender $sender, Command $cmd, string $label
                     $this->plugin->save();
                     if($this->plugin->config['bounty_new_success'] !== "") $sender->sendMessage($this->colour($this->plugin->config['bounty_new_success']));
                     foreach($this->plugin->getServer()->getOnlinePlayers() as $player){
-                        $msg = str_replace('{SENDER}', $sender->getName(), str_replace('{AMOUNT}', $amount,str_replace('{PLAYER}',$noob->getName(),$this->colour($this->plugin->config["bounty_new_broadcast"]))));
+                        $msg = str_replace('{SENDER}', $sender->getName(), str_replace('{AMOUNT}', "$amount",str_replace('{PLAYER}',$noob->getName(),$this->colour($this->plugin->config["bounty_new_broadcast"]))));
```
Comment (Lycol50) [inline src/Jack/Bounty/EventListener.php:227]: **variables are not strings**
but they are applicable when if there are 2 variables defined.
https://www.php.net/manual/en/language.variables.variable.php

File: src/Jack/Bounty/EventListener.php [hunk 2]
```diff
@@ -158,7 +158,7 @@ public function handleCommand(CommandSender $sender, Command $cmd, string $label
                     if($this->plugin->config["bounty_limit_enforced"] === true){
                         $min = $this->plugin->config["bounty_min"];
                         $max = $this->plugin->config["bounty_max"];
-                        $msg = str_replace("{MAX}", $max, str_replace("{MIN}", $min, $this->colour($this->plugin->config["bounty_new_fundlimit"])));
+                        $msg = str_replace("{MAX}", "$max", str_replace("{MIN}", "$min", $this->colour($this->plugin->config["bounty_new_fundlimit"])));
```
Comment (Lycol50) [inline src/Jack/Bounty/EventListener.php:161]: **variables are not strings**
but they are applicable when if there are 2 variables defined.
https://www.php.net/manual/en/language.variables.variable.php

**Review by Lycol50 — CHANGES_REQUESTED** [review]

File: src/Jack/Bounty/EventListener.php (hunk 1 above)
Comment (JaxkDev) [reply src/Jack/Bounty/EventListener.php:227]: What ?
This is valid syntax.

**Comment by JaxkDev** [comment]
Please re target branch to development.
Also please be consistent with casting here, you’ve used 3 different styles, “$var”, (string)$var and (string)($var) please stick with one through preferably strval()

**Comment by JaxkDev** [comment]
Inactive.

**Pull request closed without merging**